bzzeth/bzzeth_test.go

```diff
@@ -141,17 +156,18 @@ func TestBzzBzzHandshake(t *testing.T) {
 		t.Fatalf("expected no error, got %v", err)
 	}
 
-	// after handshake expect protocol to hang, peer not added to pool
-	p := b.peers.get(node.ID())
-	if p != nil {
-		t.Fatal("bzzeth swarm peer incorrectly added")
+	// Wait for sometime and see if peer is connected and idling
+	time.Sleep(1000 * time.Millisecond)
+	if _, ok := b.peers.peers[node.ID()]; !ok {
+		t.Fatalf("peer is not connected")
```

Comment:
[This test](https://travis-ci.org/ethersphere/swarm/jobs/583721259#L661) seems flaky now:

```
ERROR[09-11|15:53:03.329|github.com/ethersphere/swarm/p2p/protocols/protocol.go:238] peer.handleIncoming                      err="Message handler error: (msg code 1): received message from Swarm node"
--- FAIL: TestBzzBzzHandshake (1.09s)
    bzzeth_test.go:162: peer still connected
FAIL
coverage: 46.0% of statements
FAIL	github.com/ethersphere/swarm/bzzeth	2.364s
```

Reply:
I added some delay after sending the dummy message... It should work fine now.

Reply:
Pushed it now

Reply:
Sleep seems to be the option for now...

Reply:
I would like to hear other opinions on using sleeps.